src/test/integration/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM python:2-onbuild
+
+CMD [ "python", "./integration_tests.py", "-v", "-i", "configuration.json" ]
+
+USER nobody:nogroup
```

Comment:
We should have a dockerignore so that the configuration file is not baked into the container image.